webclient/tests/grpc/src/test/java/io/helidon/webclient/grpc/tests/GrpcClientUriTest.java

```diff
@@ -77,6 +79,7 @@ void testSupplier() {
      * Should fail to connect to first URI but succeed with second after retrying.
      */
     @Test
+    @DisabledOnOs(value = OS.WINDOWS, disabledReason = "Windows runner is too slow")
```

Comment:
Can we update the test to handle slow runners ?
Should we file an issue for this ?

Reply:
Test updated !